README.md

```diff
@@ -2,6 +2,11 @@
 Most of us study and work at the [Bio-image Analysis Technology Development group](https://physics-of-life.tu-dresden.de/bia) at the [DFG Cluster of Excellence "Physics of Life" at the TU Dresden](https://physics-of-life.tu-dresden.de/). 
 We blog about image data science, knowledge exchange and research data management in the life sciences.
 
+<hr/>
+## [GUIs: Creating graphical user interfaces with/for Python, Part III](https://biapol.github.io/blog/marcelo_zoccoler/entry_user_interfe) <small>Marcelo Zoccoler, November 29th, 2021</small>
+<img style="float: right; height:100px; width:100px" src="images/qt_logo.png">
```

Comment:
Do you know where I can find a napari logo image to put here instead of qt? @haesleinhuepf